app/models/search/query_result.rb

```diff
@@ -27,8 +27,13 @@ def items
 
   # Laying some groundwork for making better use of search results
   def decorate_items(items)
```

Comment:
Gotcha. The SOW specifically asked us to set up a decorator, in the style of that one, but if I should add them to the CollectionQuery class, I can. That definitely makes sense.

Reply:
@sarken Any thoughts on this?

Reply:
I've checked with ticking to clarify what needed to be done here, and she says:

>The `PseudDecorator` is used for pseud searches because it's referenced in the `PseudQuery` file; to make the `CollectionDecorator` be used for collection searches, it needs to be referenced in the `CollectionQuery` file. 
>
>(We probably should remove the `decorate_items` method, to prevent this sort of confusion.)

So if you can make those changes, that would be great!

Reply:
I noticed something odd when I tried this branch out on my dev environment, and I suspect it might be related to the decorator, but I'm not sure. When I look at the blurbs on the collection page, I don't see the count of works, but rather the path to the collection's works page:

<img width="1082" alt="Collection blurbs. Each blurb displays a partial URL rather than a count for the Works stat." src="https://user-images.githubusercontent.com/907055/99869863-4fe17400-2b9c-11eb-90ff-302eb561fa46.png">

This is the line of code:
https://github.com/littlelines/otwarchive/blob/4c6acff02cb452884b1549f574a5dea8875d9ca8/app/views/collections/_collection_blurb.html.erb#L51